.github/workflows/trigger-qa-build.yml

```diff
@@ -6,16 +6,78 @@ on:
       workflow:
         description: 'Desired Workflow'
         required: true
-        default: 'LernaPublish'
+        default: 'Lerna Publish'
```

Comment:
Should we actually mention lerna? If, for some reason, we don't use lerna magic anymore, we'd have to update all of this just to remove the mention of lerna. 

Also the word "Publish" is gonna scare people so I'd keep this clear and say "Pre-release". Even tho you got here from "Trigger prerelease build", we can also assume that people would be linked to this page.

I think we can generalize this by just saying

- Create a pre-release
  - Pre-release (auto determine)
    - _Convey that this is what people should be using until it fails_
  - Force Pre-release Components
  - Force Pre-release Design
  - Force Pre-release Hooks
  - Force Pre-release Design and Hooks
  - Force Pre-release All